packages/types/src/lib/interfaces.ts

```diff
@@ -457,10 +418,48 @@ export interface JsonEncryptionRetrieveRequest extends JsonAccsRequest {
   toDecrypt: string;
 }
 
-export type ExecuteJsProps = JsonExecutionRequest & {
-  // A boolean that defines if debug info will be returned or not.
-  debug?: boolean;
-};
+export interface JsonExecutionSdkParamsTargetNode
+  extends JsonExecutionSdkParams {
+  targetNodeRange: number;
+}
+
+export interface JsonExecutionSdkParams {
+  // An object that contains params to expose to the Lit Action.  These will be injected to the JS runtime before your code runs, so you can use any of these as normal variables in your Lit Action.
+  jsParams?: any;
+
+  // JS code to run on the nodes
+  code?: string;
+
+  // The IPFS ID of some JS code to run on the nodes
+  ipfsId?: string;
+
+  // the session signatures to use to authorize the user with the nodes
+  sessionSigs?: any;
+
+  // whether to run this on a single node or many
+  // targetNodeRange?: number;
+
+  // auth methods to resolve
+  authMethods?: AuthMethod[];
+}
+
+export interface JsonExecutionRequestTargetNode extends JsonExecutionRequest {
+  targetNodeRange: number;
+}
+
+export interface JsonExecutionRequest {
+  authSig: AuthSig;
```

Comment:
AuthSig is still present

Reply:
https://github.com/LIT-Protocol/js-sdk/pull/444#discussion_r1584883211